lib/plugins/bayesian-optimization/model/bayesian_optimization_model_types.dart

```diff
@@ -0,0 +1 @@
+enum BayesianOptimizationModelTypes { gaussian_process }
```

Comment:
```suggestion
enum BayesianOptimizationModelType { gaussian_process }
```
It is usually better to declare enums in singular, see e.g. https://stackoverflow.com/questions/15755955/naming-of-enums-in-java-singular-or-plural